azure-pipelines.yml

```diff
@@ -9,9 +9,12 @@ trigger:
     - 'doc/*'
 
 pr:
-- master
-- 1.2.x
-- 1.3.x
+  autoCancel: true
+  branches:
+    include:
+    - master
+    - 1.3.x
+    - 1.2.x
```

Comment:
Unrelated, but could you remove 1.2.x from this list. I forgot to do this in my other PR.